understanding/20/use-of-color.html

```diff
@@ -139,13 +156,7 @@ <h2>Benefits of Use of Color</h2>
             in other visual ways.
          </li>
          
-         <li>People using limited color monochrome displays may be unable to access
-            color-dependent information.
-         </li>
-         
-         <li>Users who have problems distinguishing between colors can look or listen for text
-            cues.
-         </li>
+         <li>Users who have problems distinguishing between colors can look for other visual cues.</li>
```

Comment:
Not a problem, but in points 1 and 2, we delineate the beneficiaries of a proper implementation. In points 3 and 4, alongside identifying the beneficiaries, we also clarify that they will reap the benefits if other visual cues are present (although this may not be necessary). Should we stay consistent?

Reply:
i think with the first two the only thing you could then say is a platitude/truism a la ", so not relying on color alone will help them". in a sense, the first three bullets all get to the same point and could even be combined.